src/division.py

```diff
@@ -0,0 +1,17 @@
+def main():
+
+    # takes numerator and denominator from the user
+    numerator = int(input("Enter the numerator: "))
+    denominator = int(input("Enter the denominator: "))
+
+    # try and except block to handle the case if user enters 0
+    try:
+        divide = (numerator / denominator)  # do the division
+    except ZeroDivisionError:
+        print("oops, you have entered denominator as 0, can't divide by 0")
+    else:
+        print(divide)  # print the final division
```

Comment:
Don't just print variable. Put a meaningful message. like...

```
print("Your final division is: %s", divide)
```

Reply:
got it i will solidify the script